code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -127,25 +154,70 @@
 	return (BRUTELOSS)
 
 
-/obj/item/weapon/scalpel/laser
-	heat_production = 0
+/obj/item/weapon/scalpel/laser //Not obtainable normally. More to just be a parent to laser scalpels. Three of them were intended it seems but one was dropped so I used the middle tier sprite here.
+	name = "laser scalpel"
+	desc = "A laser scalpel."
+	icon_state = "scalpel_laser2old"
+	item_state = "laserscalpel2old"
 	damtype = "fire"
 	sharpness_flags = SHARP_TIP | SHARP_BLADE | HOT_EDGE
+	surgery_speed = 0.5
 	var/cauterymode = 0 //1 = cautery enabled
+	var/obj/item/weapon/cautery/laser/held
+
+/obj/item/weapon/scalpel/laser/New()
+	..()
+	icon_state = "scalpel_laser2old_off"
+	held = new /obj/item/weapon/cautery/laser(src)
 
 /obj/item/weapon/scalpel/laser/attack_self(mob/user)
-	if(!cauterymode)
+	if(!cauterymode && held)
 		to_chat(user, "You disable the blade and switch to the scalpel's cautery tool.")
 		heat_production = 1600
 		sharpness = 0
 		sharpness_flags = 0
+	else if(!held)
+		to_chat(user, "\The [src] lacks a cautery attachment.")
+		return
 	else
 		to_chat(user, "You return the scalpel to cutting mode.")
 		heat_production = 0
 		sharpness = initial(sharpness)
 		sharpness_flags = initial(sharpness_flags)
 	cauterymode = !cauterymode
 
+/obj/item/weapon/scalpel/laser/examine(mob/user)
+	..()
+	if(!cauterymode)
+		to_chat(user, "\The [src] is in cutting mode.")
+	else
+		to_chat(user, "\The [src] is in cautery mode.")
+
+/obj/item/weapon/scalpel/laser/attackby(var/obj/item/used_item, mob/user)
+	if(isscrewdriver(used_item) && cauterymode)
+		if(held)
+			to_chat(user, "<span class='notice'>You detach \the [held] and \the [src] switches to cutting mode.</span>")
+			playsound(get_turf(src), "sound/items/screwdriver.ogg", 10, 1)
+			held.add_fingerprint(user)
+			held.forceMove(get_turf(src))
+			held = null
+			heat_production = 0
+			sharpness = initial(sharpness)
+			sharpness_flags = initial(sharpness_flags)
+			cauterymode = 0
+	else if(istype(used_item, /obj/item/weapon/cautery/laser))
+		if(held)
+			to_chat(user, "<span class='notice'>There's already a cautery attached to \the [src].</span>")
+			return
```

Comment:
You can negate the need for this `return` by changing the second `if()` to an `else if()`.